website/screens/components/flex/usage/FlexUsagePage.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  DxcFlex,
+  DxcParagraph,
+  DxcBulletedList,
+} from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+
+const sections = [
+  {
+    title: "Axes",
+    content: (
+      <>
+        <DxcParagraph>
+          Flexboxes are comprised mainly of two axes: the main axis and the
+          cross axis. The main axis is defined by the flex-direction property,
+          and the cross axis runs perpendicular to it.
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            The main axis is defined by flex-direction, which has four possible
+            values: <Code>row</Code>, <Code>row-reverse</Code>,{" "}
+            <Code>column</Code> and <Code>column-reverse</Code>.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            The cross axis runs perpendicular to the main axis.
+            <DxcBulletedList type="circle">
+              <DxcBulletedList.Item>
+                If the flex-direction (main axis) is set to row or row-reverse
```

Comment:
In the line 24 it is used the component `Code` for the values. I would continue use it in the rest when referring to this values as in this line 31.

Reply:
I agree in the others, but not here. In the Axes section we are defining the possible values of `direction` prop, so it makes sense to remark them with the `Code`. But here we are only talking about it, I think it makes no sense to remark it. Like in the Flexbox container section, that we are not remarking it.

Reply:
Done